docs/management/connectors/action-types/xmatters.asciidoc

```diff
@@ -0,0 +1,113 @@
+     url: https://test.host?apiKey=1234-abcd
+--
+
+Config defines information for the connector type.
+
+`url`:: A URL string that corresponds to *URL*. Only used if `usesBasic` is true.
+
+`usesBasic`:: A boolean that corresponds to *Authentication Type*. If `true`, this connector will require values for `user` and `password` inside the secrets configuration. Defaults to `true`.
+
+Secrets defines sensitive information for the connector type.
+
+`user`:: A string that corresponds to *User*. Required if `usesBasic` is set to `true`.
+
+`password`:: A string that corresponds to *Password*. Should be stored in the <<creating-keystore, {kib} keystore>>. Required if `usesBasic` is set to `true`.
+
+`url`:: A URL string that corresponds to *URL*. Only used if `usesBasic` is false, indicating the API Key is included in the URL.
+
+[float]
+[[define-xmatters-ui]]
+==== Define connector in Stack Management
+
+Define xMatters connector properties. Choose between Basic and URL authentication for the requests.
+
+[role="screenshot"]
+image::management/connectors/images/xmatters-connector-basic.png[xMatters connector with basic authentication]
+
+[role="screenshot"]
+image::management/connectors/images/xmatters-connector-url.png[xMatters connector with url authentication]
+
+Test xMatters rule parameters.
+
+[role="screenshot"]
+image::management/connectors/images/xmatters-params-test.png[xMatters params test]
+
+[float]
+[[xmatters-action-configuration]]
+==== Action configuration
+
+xMatters rules have the following properties.
+
+Severity::                                                     Severity of the rule.
+Tags:: Comma-separated list of tags for the rule as provided by the user in Elastic.
+
+[float]
+[[xmatters-benefits]]
+==== Configure xMatters
+
+By integrating with xMatters, you can:
+
+. Leverage schedules, rotations, escalations, and device preferences to quickly engage the right resources.
+. Allow resolvers to take immediate action with customizable notification responses, including incident creation.
+. Reduce manual tasks so teams can streamline their resources and focus.
+
+[float]
+[[xmatters-connector-prerequisites]]
+==== Prerequisites
+To use the Elastic xMatters connector either install the Elastic workflow template, or add the Elastic Alerts trigger to one of your existing xMatters flows. Once the workflow or trigger is in your xMatters instance, configure Elastic to send alerts to xMatters.
+
+[role="screenshot"]
+image::management/connectors/images/xmatters-copy-url.png[Elastic Trigger in xMatters]
+
+. In xMatters, double-click the Elastic trigger to open the settings menu.
+. Choose the authentication method and set your authenticating user.
+. Copy the Initiation URL.
+. In Elastic, open the xMatters connector.
+. Set the authentication method, then paste the Initiation URL.
+
+Note: if using Basic authentication, make sure you use the Web / App Login ID. This value can be found in the Edit Profile modal for each user.
```

Comment:
I think we should be clearer about where we're getting these IDs from and where we're using them. For example:
```suggestion
Note: If you use basic authentication, specify the Web / App Login ID in the user credentials for the connector. This value can be found in the Edit Profile modal in xMatters for each user.
```